x-pack/plugins/synthetics/server/synthetics_service/project_monitor/normalizers/http_monitor.test.ts

```diff
@@ -130,9 +130,9 @@ describe('http normalizers', () => {
           errors: [
             {
               details:
-                'Multiple urls are not supported for http project monitors in 8.5.0. Please set only 1 url per monitor. You monitor was not created or updated.',
+                'http project monitors must have exactly one value for field `urls` in version `8.5.0`. Your monitor was not created or updated.',
```

Comment:
Hey @afgomez, I know this is for testing, but do you know if the values used instead of variables here are correct?
If that is in the scope of this PR or another one, I'd like to suggest some tweaks, for example:

- Acronyms should be uppercase: HTTP, URLs
- {key} here is 'urls'. If we expect only 1 value for some fields, these field names should be singular (in addition to uppercase for acronyms): `URL`

I'm happy to take a look as/when needed.

Reply:
Let me answer to both:

> Acronyms should be uppercase: HTTP, URLs

This is tricky. Although `http` is indeed an acronym, is also what the user writes when they call the API (`type: http`, `type: browser`, ...) so we cannot blindly turn it to uppercase. One thing I can do to get around it is to put it in between quotes like I do with the field:

```
`http` project monitors must have....
```

> {key} here is 'urls'.

The reasoning here is similar. I'm quoting the parameter that the users use when calling the API. I agree it should be called `url` since we only accept one, but I guess there's some reason why we need to keep it as `urls`.

```
type: browser
urls: ['...']
```